packages/frontend/workspace/src/blob/storage/indexeddb.ts

```diff
@@ -0,0 +1,32 @@
+import { createStore, del, get, keys, set } from 'idb-keyval';
+
+import type { BlobStorage } from '../engine';
+
+export const createIndexeddbBlobStorage = (
```

Comment:
may need blocksuite team's awareness that we moved idb's blob storage into affine.
@doodlewind